ClassTypes/PowerShell/EthicalHacking/Create-EthicalHackingLabplanAndLab.ps1

```diff
@@ -44,20 +57,23 @@ trap {
 #
 
 # Download AzLab module file, import, and then delete the file
+if (-not (Get-Module -ListAvailable -Name 'Az')) {
+    Import-Module Az -Force 
+}
 
-Import-Module Az.LabServices -Force
+$ClassName = "EthicalHacking"
```

Comment:
I also noticed this was a parameter before, but now it's hard-coded in the script...  I think either way is probably fine, but just curious on the reason?  I also think maybe Location could be handled the same was as ClassName (as parameter or hard coded)?